src/main/java/com/rocketden/main/service/SubmitService.java

```diff
@@ -13,17 +16,33 @@
 @Service
 public class SubmitService {
 
-    private final RoomRepository repository;
     private final SocketService socketService;
 
-    protected SubmitService(RoomRepository repository, SocketService socketService) {
-        this.repository = repository;
+    protected SubmitService(SocketService socketService) {
```

Comment:
Why isn't there an `@Autowired` attribute above this constructor? (I'm also not sure how it works without it, do we not need it on the other classes?)

Reply:
My bad, I left it out on accident - I believe when using constructor injection as we have been doing, we don't need the autowired annotation, although I guess it helps with readability and all

Reply:
No problem, thanks for adding it back, we can always reconsider adding that `@Autowired` tag if it doesn't actually do anything but for now this sounds good!